pandas/core/base.py

```diff
@@ -929,11 +929,17 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
         """
         Return an ndarray of the maximum argument indexer.
```

Comment:
is this sentence accurate?  it looks like we return an int, not ndarray

Reply:
(also relevant for the Returns section)

Reply:
Do you think this okay?
```
Return row position of the maximum argument indexer.
```

And also change for the Returns section:
```
Returns
-------
int
    row position of the maximum values.
```

Reply:
"maximum argument indexer" is awkward.  how about "Returns int position of the largest value in the Series"

Reply:
> 
> 
> "maximum argument indexer" is awkward. how about "Returns int position of the largest value in the Series"

I think this one is easier to understand.